tools/shell/linenoise.cpp

```diff
@@ -139,25 +139,24 @@ static int history_len = 0;
 static char **history = NULL;
 static char *history_file = NULL;
 #ifndef DISABLE_HIGHLIGHT
-#include <string>
 
 static int enableHighlighting = 1;
 struct Color {
 	const char *color_name;
 	const char *highlight;
 };
 static Color terminal_colors[] = {{"red", "\033[31m"},           {"green", "\033[32m"},
-                                  {"yellow", "\033[33m"},        {"blue", "\033[34m"},
-                                  {"magenta", "\033[35m"},       {"cyan", "\033[36m"},
-                                  {"white", "\033[37m"},         {"brightblack", "\033[90m"},
-                                  {"brightred", "\033[91m"},     {"brightgreen", "\033[92m"},
-                                  {"brightyellow", "\033[93m"},  {"brightblue", "\033[94m"},
-                                  {"brightmagenta", "\033[95m"}, {"brightcyan", "\033[96m"},
-                                  {"brightwhite", "\033[97m"},   {nullptr, nullptr}};
-static std::string bold = "\033[1m";
-static std::string keyword = "\033[32m\033[1m";
-static std::string constant = "\033[33m";
-static std::string reset = "\033[00m";
+								{"yellow", "\033[33m"},        {"blue", "\033[34m"},
+								{"magenta", "\033[35m"},       {"cyan", "\033[36m"},
+								{"white", "\033[37m"},         {"brightblack", "\033[90m"},
+								{"brightred", "\033[91m"},     {"brightgreen", "\033[92m"},
+								{"brightyellow", "\033[93m"},  {"brightblue", "\033[94m"},
+								{"brightmagenta", "\033[95m"}, {"brightcyan", "\033[96m"},
+								{"brightwhite", "\033[97m"},   {nullptr, nullptr}};
+char const *bold = "\033[1m";
```

Comment:
Could you change this to `const char` for consistency with the rest of the codebase?

Reply:
Done.